letsencrypt-auto-source/letsencrypt-auto

```diff
@@ -1124,12 +1127,14 @@ else
   # package). Phase 2 checks the version of the locally installed certbot.
 
   if [ ! -f "$VENV_BIN/letsencrypt" ]; then
-    if [ "$HELP" = 1 ]; then
-      echo "$USAGE"
-      exit 0
+    if [ -z "$OLD_VENV_PATH" -o ! -f "$OLD_VENV_PATH/bin/letsencrypt" ]; then
```

Comment:
Under what circumstances would `[ -z "$OLD_VENV_PATH" ]` be meaningful? Did you mean something like `! -d`?